fuzzing_test.go

```diff
@@ -87,25 +84,32 @@ func TestFuzz(t *testing.T) {
 
 	fuzz := fuzz.New()
 
-	for i := 0; i < rand.Intn(*numCases)+10; i++ {
-		n := rand.Intn(*numEvents) + 10
-		events := make([]termbox.Event, n)
+	min := func(a, b int) int {
+		if a < b {
+			return a
+		}
+		return b
+	}
+
+	// number of events in tcell.SimulationScreen is limited 10
```

Comment:
`*numCases` doesn't seem to be related to the limitation of the number of events, is it right?

Reply:
tcell simulation screen limit 10 events https://github.com/gdamore/tcell/blob/master/simulation.go#L112

Reply:
Oh, that's right🙏